templates/webapi/layouts/bootstrap.html.ep

```diff
@@ -29,6 +29,7 @@
           %= content_for 'head_javascript'
           $(function() {
             setupForAll();
+            backToTop();
```

Comment:
Where does this function come from? I've tested it locally and got:

```
jquery-3.3.1.js:3827 Uncaught ReferenceError: backToTop is not defined
    at HTMLDocument.<anonymous> ((index):64)
    at mightThrow (jquery-3.3.1.js:3534)
    at process (jquery-3.3.1.js:3602)
```

Reply:
Ah, yea, my latest force push was completely busted. No idea what happened.